mkosi.md

```diff
@@ -365,6 +365,11 @@ details see the table below.
 : If specified, mkosi does not build unified kernel images and instead installs kernels with a separate
   initrd and boot loader config to the efi or bootloader partition.
 
+`--hostonly`
```

Comment:
```suggestion
`--hostonly-initrd`
```
This would make it clearer, without reading the docs, what this is for. It would need renaming of `args.hostonly` in either the whole PR or at least the argparse bits and docs, though.